packages/plugins/time/package.json

```diff
@@ -9,9 +9,10 @@
     "build": "codegen-templates-scripts build",
     "pretest": "yarn build"
   },
-  "dependencies": {},
+  "dependencies": {
+    "moment": "2.22.2"
```

Comment:
I added it to dependencies and also did set it in webpack config as external module.